site/source/components/SimulateurWarning.tsx

```diff
@@ -80,14 +84,15 @@ export default function SimulateurWarning({
 						</Li>
 					</>
 				)}
-				{['indépendant', 'profession-libérale'].includes(simulateur) && (
-					<Li>
-						<Trans i18nKey="simulateurs.warning.année-courante">
-							Le montant calculé correspond aux cotisations de l’année 2022
-							(pour un revenu 2022).
-						</Trans>
-					</Li>
-				)}
+				{['indépendant', 'profession-libérale'].includes(simulateur) &&
+					(année == null || année == 2022) && (
+						<Li>
+							<Trans i18nKey="simulateurs.warning.année-courante">
+								Le montant calculé correspond aux cotisations de l’année 2022
```

Comment:
```suggestion
								Le montant calculé correspond aux cotisations de l’année {année}
```

On aimerait afficher l'avertissement également sur 2021. Normalement, l'interpolation marche toute seule dans le composant `Trans` (à vérifier).